src/Angor/Client/Pages/Settings.razor

```diff
@@ -707,8 +707,12 @@
 
     private async Task AddExplorer()
     {
-        if (!string.IsNullOrWhiteSpace(newExplorerLink))
-        {
+        if (string.IsNullOrWhiteSpace(newExplorerLink))
+            {
```

Comment:
what is going on with the braces? they need to be under the if, look at the rest of the code to see the formatting